src/layouts/InterfaceLayout/InterfaceLayout.vue

```diff
@@ -431,6 +431,7 @@ export default {
         .getBlockNumber()
         .then(res => {
           this.blockNumber = res;
+          this.$store.dispatch('updateBlockNumber', res);
```

Comment:
doesnt interfacelayout pass blocknumber as a prop?

Reply:
Yeah but the component is outside of it. Remember the double polling that was removed a couple of releases ago?